enterprise/app/controllers/api/v1/accounts/agent_capacity_policies/inbox_limits_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::V1::Accounts::AgentCapacityPolicies::InboxLimitsController < Api::V1::Accounts::EnterpriseAccountsController
+  before_action :fetch_policy
```

Comment:
rough notes:

agent_capacity

```
name:
description:
...
```

agent_capacity/inboxes
```
inbox: {
}
limit: 
```

URL: `/api/v1/accounts/:account_id/agent_capacity_policies/:agent_capacity_policy_id/users`

Get - should return all the users.
POST- should return the user which was added
UPDATE - user
DELETE - ok

agent_capacity/users
users: